### PR TITLE
Add hunger, speed, helditem, fps, tps options to DiscordRPC

### PR DESCRIPTION
**Pull request by Xiaro**
**Describe the pull**
As what the title says

**Describe how this pull is helpful**
Closing issue #1008

### CODE REVIEW

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt [hunk 1]
```diff
@@ -30,18 +34,23 @@ class DiscordRPC : Module() {
...
             LineInfo.VERSION -> KamiMod.MODVERSMALL
             LineInfo.WORLD -> if (mc.isIntegratedServerRunning) "Singleplayer" else if (mc.getCurrentServerData() != null) "Multiplayer" else "Main Menu"
-            LineInfo.DIMENSION -> InfoCalculator.playerDimension(mc)
+            LineInfo.DIMENSION -> playerDimension(mc)
             LineInfo.USERNAME -> if (mc.player != null) mc.player.name else mc.getSession().username
             LineInfo.HEALTH -> if (mc.player != null) mc.player.health.toInt().toString() + " hp" else "No hp"
+            LineInfo.HUNGER -> if (mc.player != null) mc.player.getFoodStats().foodLevel.toString() + " hunger" else "No hunger"
             LineInfo.SERVER_IP -> if (mc.getCurrentServerData() != null) mc.getCurrentServerData()!!.serverIP else if (mc.isIntegratedServerRunning) "Offline" else "Main Menu"
             LineInfo.COORDS -> if (mc.player != null && coordsConfirm.value) "(" + mc.player.posX.toInt() + " " + mc.player.posY.toInt() + " " + mc.player.posZ.toInt() + ")" else "No coords"
+            LineInfo.SPEED -> if (mc.player != null) "Travling at " + speed(true, mc, 2).toString() + " Km/h" else "No speed"
```
Comment (dominikaaaa) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt:50]: Could this use the speed setting from InfoOverlay?

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt [hunk 2]
```diff
@@ -30,18 +34,23 @@ class DiscordRPC : Module() {
     var line4Setting: Setting<LineInfo> = register(Settings.e("Line2Right", LineInfo.HEALTH)) // state right
 
     enum class LineInfo {
-        VERSION, WORLD, DIMENSION, USERNAME, HEALTH, SERVER_IP, COORDS, NONE
+        VERSION, WORLD, DIMENSION, USERNAME, HEALTH, HUNGER, SERVER_IP, COORDS, SPEED, HELDITEM, FPS, TPS, NONE
```
Comment (dominikaaaa) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt:37]: `HELDITEM` > `HELD_ITEM` please

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt [hunk 3]
```diff
@@ -30,18 +34,23 @@ class DiscordRPC : Module() {
...
-            LineInfo.DIMENSION -> InfoCalculator.playerDimension(mc)
+            LineInfo.DIMENSION -> playerDimension(mc)
             LineInfo.USERNAME -> if (mc.player != null) mc.player.name else mc.getSession().username
             LineInfo.HEALTH -> if (mc.player != null) mc.player.health.toInt().toString() + " hp" else "No hp"
+            LineInfo.HUNGER -> if (mc.player != null) mc.player.getFoodStats().foodLevel.toString() + " hunger" else "No hunger"
             LineInfo.SERVER_IP -> if (mc.getCurrentServerData() != null) mc.getCurrentServerData()!!.serverIP else if (mc.isIntegratedServerRunning) "Offline" else "Main Menu"
             LineInfo.COORDS -> if (mc.player != null && coordsConfirm.value) "(" + mc.player.posX.toInt() + " " + mc.player.posY.toInt() + " " + mc.player.posZ.toInt() + ")" else "No coords"
+            LineInfo.SPEED -> if (mc.player != null) "Travling at " + speed(true, mc, 2).toString() + " Km/h" else "No speed"
+            LineInfo.HELDITEM -> "Holding " + (if (mc.player != null && !mc.player.getHeldItem(EnumHand.MAIN_HAND).isEmpty()) mc.player.getHeldItem(EnumHand.MAIN_HAND).displayName.toLowerCase() else "no item")
```
Comment (dominikaaaa) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt:51]: Capitalize N in No

Comment (Xiaro) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt:51]: It is "Holding no item" so

Comment (dominikaaaa) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt:51]: Oh I missed that.

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt (hunk 1 above)
Comment (dominikaaaa) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt:50]: Like the unit it uses, m/s vs km/h

**Pull request merged**